src/static/2cards_layout_b.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <!--Import Google Icon Font-->
+  <link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <!--Import materialize.css-->
+  <link type="text/css" rel="stylesheet" href="css/materialize.css"  media="screen,projection"/>
+  <link rel="stylesheet" type="text/css" href="css/cards.css">
+  <!--Let browser know website is optimized for mobile-->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <meta charset="utf-8">
+  <title>sdm-app</title>
+  <script language="javascript" type="text/javascript">
+  var scrt_var = window.location.href.split("=")[1] || 1;
+
+  </script>
+</head>
+<body>
+
+  <section class="hbs-container"></section>
+
+  <!-- actual client side template -->
+  <script id="template-hbs" type="text/x-handlebars-template">
+    <div class="row s12">
+      <h2>{{getQuestion questions}}</h2>
+      <div class="col s12 m8 l8">
+        <div class="card horizontal">
+          <div class="card-image-duo">
+            <div class="card-name">
+              <h4>{{treatments.practical_support.name}}</h4>
+            </div>
+            <img src={{treatments.practical_support.icon}} alt="" />
+          </div>
+          <div class="card-image-duo">
+            <div class="card-name">
+              <h4>{{treatments.talking_therapy.name}}</h4>
+            </div>
+            <img src={{treatments.talking_therapy.icon}} alt="" />
+          </div>
+          <div class="card-stacked-duo">
+            <div class="card-content">
+              <p>{{getPracticalSupport questions}}</p>
+            </div>
+            <div class="card-action">
+              <a href="#">This is a link</a>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="col s12 m4 l4">
+        <div class="card horizontal">
+          <div class="card-image">
+            <div class="card-name">
+              <h4>{{treatments.medication.name}}</h4>
+            </div>
+            <img src={{treatments.medication.icon}} alt="" />
+          </div>
+          <div class="card-stacked">
+            <div class="card-content">
+              <p>{{getTalkingTherapy questions}}</p>
+            </div>
+            <div class="card-action">
+              <a href="#">This is a link</a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <!--  <a href="3cards_layout.html" onclick="location.href=this.href+'?key='+ ++scrt_var;return false;">Go to the next question</a>-->
+      <a href="3cards_layout.html" onclick='location.href="file:///home/nori/FAC/sdm-app/src/static/"+"2cards_layout_b.html?key="+ ++scrt_var;return false;'>Go to the next question</a>
```

Comment:
I'm addig a comment explaining what I do here and why I am doing that. I agree that this is ugly. I considered using an eventListener...but I couldn't figure out how I can keep track of the question id.